publication-commons/src/main/java/no/unit/nva/publication/service/impl/DynamoDBPublicationService.java

```diff
@@ -144,7 +149,7 @@ private void allFieldsAreNonNull(String owner, URI publisherId, String authoriza
             publicationSummary = objectMapper.readValue(item.toJSON(), PublicationSummary.class);
             return Optional.of(publicationSummary);
         } catch (JsonProcessingException e) {
-            Logger.logError(e);
+            System.out.println(e.getMessage());
```

Comment:
We stopped using the logger?

Reply:
That Logger was just a wrapper for System.out.println(). Changed it now to throw exception with message up to the nva-commons handler and the LambdaLogger.